src/RadhydroSimulation.hpp

```diff
@@ -857,6 +869,14 @@ void RadhydroSimulation<problem_t>::advanceHydroAtLevelWithRetries(int lev, amre
 		amrex::Copy(originalFineData, fineData, 0, 0, fineData.nComp(), 0);
 	}
 
+#ifdef AMREX_PARTICLES
```

Comment:
I don't understand the logic for using the AMREX_PARTICLES guards here. Should this be controlled by physics traits like everything else? Why use ifdef's here instead?

Reply:
It's possible to compile AMReX without particle support. That might conceivably be useful for debugging (e.g., compiler bugs) in the future, so I'd like to keep it with the ifdef here. Otherwise, it could just be controlled with a runtime parameter.

Reply:
OK, that's fine.